app/components/Records/Record.js

```diff
@@ -15,9 +17,9 @@ class Record extends Component {
     const currentTypeIndex = DROPDOWN_TYPES.findIndex((d) => d.label === type);
 
     return (
-      <TableRow>
-        <TableItem>{DROPDOWN_TYPES[currentTypeIndex].label}</TableItem>
-        <TableItem>{value}</TableItem>
+      <TableRow className={classNames(this.props.className)}>
```

Comment:
```suggestion
      <TableRow className={this.props.className}>
```

Reply:
`classNames()` actually serves a purpose here - it would default `undefined` to `''`

Reply:
Oh, I see. Passing `className={undefined}` should be fine because it's optional.

But `TableRow` should use the `classnames` and threat the property as optional
https://github.com/kyokan/bob-wallet/blob/5f6f352a58e320ee4b4bfa8ca62979247958455f/app/components/Table/index.js#L58